recipes/antlr-python-runtime/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "antlr4-python2-runtime" %}  # [py2k]
+{% set sha256 = "3bc09f7fe058fe8ede92e5d292f7a6932022cde28bb0ba5f44f96842c3c894d8" %}  # [py2k]
+
+{% set name = "antlr4-python3-runtime" %}  # [py3k]
+{% set sha256 = "1b26b72c4492cef310542da10bf6b2ab4aa1775618fc6003f75b55ae9eaa3fd3" %}  # [py3k]
+
+{% set version = "4.7.1" %}
+
+
+package:
+  name: antlr-python-runtime
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - antlr4
+    - antlr4.atn
+    - antlr4.dfa
+    - antlr4.error
+    - antlr4.tree
+    - antlr4.xpath
+
+about:
+  home: http://www.antlr.org
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Can you add the license file on a new PR? We missed that.

Reply:
Skimmed through the `sdist`s and did not find one. Can you please raise an issue upstream to get the license file included in future `sdist`s?